guides/V1_ActionsEvents.md

```diff
@@ -0,0 +1,183 @@
+
+ - You should display a continue to space button to complete the process.
+
+ Let's walk through these.
+
+### Protect the explicit action endpoint
+
+Making an endpoint publicly available has security risks; for example, your endpoint could receive requests from malicious sources. To minimize risk:
+- Be prepared for <a href="https://en.wikipedia.org/wiki/Network_security#Types_of_Attacks" target="_blank">network attacks</a> and ensure that appropriate counter-measures are in place.
+- Do not assume that the request or its parameters are trustworthy until you have verified the call.
+
+
+### Verify the call
+
+Use the url parameter `actionToken` and utilize Watson Work Services to verify the call.
+Authorize your app [as an App](../references/V1_oauth_token_client_credentials.yml) and access the action data with the `actionToken` - see [Get Action Context](./V1_GetActionContext.md).
+If the access to the action data is successful, then you can be sure that the call has been triggered by Watson Workspace.
+You will also get the action context data, which may include things like `spaceId` and `userId`.  `spaceId` is the unique ID of the space for which the action has been triggered.  While `userId` is the unique ID of the user who triggers the action.
+
+<div class="tip">
+  <img src="https://github.com/watsonwork/watsonwork-developer-docs/images/note_pencil.png" />
+  <p><strong>Note:</strong> If the implementation of the your explicit action handler page is still based on a version prior to 1.1.0 of the <a href="https://developer.watsonwork.ibm.com/docs/apps/get-action-context>Get Actiopn Context API</a>, then you have to do the oauth flow to verify the caller as a user of Watson Work Services, to obtain the user id and name, and to establish a link between the user's ID and the user's name.</p>
+</div>
+
+
+### Protect the user's sensitive data
+
+You can utilize Watson Work Services to authorize the user as a valid Watson Work Services user by [getting the user details](https://developer.watsonwork.ibm.com/docs/people/get-user-information) and / or you can also authorize the user against your own credential store of users authorized to use your app.
+
+
+### Authorizing the app
+
+Your app must be authorized to request services from Watson Work Services. There are two ways to authorize an app; both require the App ID and App Secret:
+- [as an App](https://developer.watsonwork.ibm.com/docs/api-reference/authenticate-as-an-app) can retrieve details from the app perspective (for example, the names of spaces that the app was added to)
+- [on behalf of a User](https://developer.watsonwork.ibm.com/docs/api-reference/authorize-on-behalf-of-a-user) can retrieve details from the user perspective (for example, the names of spaces that the user is a member of)
+
+
+<a id="get-the-user-id-and-name"></a>
+
+### Get the user ID and name
```

Comment:
Not sure this section is needed.  This was the old config flow whereby you were not guaranteed who the user was.  This data will be provided based on the type of action.